storage/storage-interface/src/async_proof_fetcher.rs

```diff
@@ -76,6 +78,9 @@ impl AsyncProofFetcher {
 
     // Schedules proof reading work in a background running thread pool.
     fn schedule_proof_read(&self, state_key: StateKey, version: Version) {
+        let _timer = TIMER
```

Comment:
According to my previous observation schedule_proof_read is fast, so the timer here may contribute significant portion of time .

Reply:
let's take a look and PR++ to remove this later :trollface: